src/rootPages/Designer/editors/views/ABViewTab.js

```diff
@@ -40,7 +40,7 @@ export default function (AB) {
             const ids = this.ids;
             const baseView = this.view;
             const component = this.component;
-            const componentUI = this.component.ui();
+            const componentUI = this.component.ui().rows[0];
```

Comment:
should we have a check to make sure `this.component.ui()` returns a valid entry?

`const componentUI = this.component.ui()?.rows[0];`